analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CheckArgumentException.cs

```diff
@@ -91,12 +91,19 @@ void Foo5(int a)
             throw new DuplicateWaitObjectException(paramName); // Compliant
         }
 
-        void Foo6(int a)
+        void Foo6(int a, int @b)
         {
             throw new ArgumentNullException(nameof(a));
             throw new ArgumentNullException(nameof(Foo5)); // Noncompliant
             throw new ArgumentOutOfRangeException(nameof(Foo5)); // Noncompliant
             throw new DuplicateWaitObjectException(nameof(Foo5)); // Noncompliant
+
+            throw new ArgumentNullException("a");
```

Comment:
```suggestion
            throw new ArgumentNullException("a");
            throw new ArgumentNullException("@a");
```

Reply:
The second one is raising an issue but I would not consider it as a FP.